introduction.md

```diff
@@ -1 +1,6 @@
 My name is Ben and my favorite CodeSchool path is JavaScript
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?